### PR TITLE
#trivial Feature tests

### PR DESCRIPTION
**Pull request by klzns**
#### What is the purpose of this pull request?

Add snapshot tests to Button so we can keep track of changes in style and avoid breaking stores' customizations.

Motivation of this PR is to guarantee that #1068 doesn't break our Button.

> **There's no changelog change because it doesn't change our API**

#### How should this be manually tested?

Run tests 😅 

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

- [x] Changes in tests

### CODE REVIEW

File: react/components/Alert/index.test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { render, fireEvent } from 'react-testing-library'
 
 import Alert from './index'
 
-describe('Alert', () => {
+describe.skip('Alert', () => {
```
Comment (iaronaraujo) [inline react/components/Alert/index.test.js]: Why the skip?

Comment (klzns) [reply react/components/Alert/index.test.js]: image

and

image

Comment (iaronaraujo) [reply react/components/Alert/index.test.js]: There should be a comment above the skip then

Comment (iaronaraujo) [reply react/components/Alert/index.test.js]: Add a comment above the skip

**Pull request merged**